gunicorn.py

```diff
@@ -200,15 +200,16 @@ def worker_int(worker):
     worker.log.info("worker received INT or QUIT signal")
 
     # get traceback info
-    import threading, sys, traceback
+    import sys
+    import threading
+    import traceback
+
     id2name = dict([(th.ident, th.name) for th in threading.enumerate()])
     code = []
     for threadId, stack in sys._current_frames().items():
-        code.append("\n# Thread: %s(%d)" % (id2name.get(threadId,""),
-            threadId))
+        code.append("\n# Thread: %s(%d)" % (id2name.get(threadId, ""), threadId))
```

Comment:
Is there a reason to not use .format() here? Feel like it would be cleaner and consistent with everywhere else.

Reply:
I might be wrong but I think we copy/pasted the guincorn.py file from somewhere.
If we're wanting to refactor/tidy that file it's probably something to do in another PR. I just fixed the things that flake8 was complaining about here